src/http/get-micropub/config/contact.js

```diff
@@ -0,0 +1,25 @@
+const arc = require('@architect/functions')
+
+async function contact (filter = '') {
+  if (filter) {
+    filter = filter.toLowerCase().replace(/[^a-z0-9-]/, '')
+  }
+  const data = await arc.tables()
+  const results = await data.contacts.scan({})
+  const contacts = results.Items.map(c => {
+    if (filter) {
+      if (c.nickname.startsWith(filter)) {
+        return c
+      } else {
+        return null
+      }
```

Comment:
This shouldn't be needed, as it'd return `null` / `undefined` anyway?

```suggestion
```

Reply:
I get the following from Standard.js without returning the null: 
> Array.prototype.map() expects a value to be returned at the end of arrow function. (array-callback-return)

Reply:
Interesting - fair enough!